api/v0/finder/model/model.go

```
@@ -56,9 +57,11 @@ func (pr ProviderResult) Equal(other ProviderResult) bool {
 }
 
 func ProviderResultFromValue(value indexer.Value, addrs []multiaddr.Multiaddr) ProviderResult {
+	metadata, _ := indexer.DecodeMetadata(value.MetadataBytes)
```

do we want to ignore this error?